packages/keystone/README.md

```diff
@@ -6,14 +6,41 @@ order: 1
 
 # keystone
 
+## Constructor
+
+### Usage
+
+```javascript
+const keystone = new Keystone({
+  /*...config */
+});
+```
+
+### Config
+
+| Option                  | Type       | Default    | Description                                                                                                                                                 |
```

Comment:
Can we remove `FOO` from all of these? I'll try and help with some descriptions today if I get time :)